src/content/en/2020/accessibility.md

```diff
@@ -8,17 +8,437 @@ authors: [oluoluoxenfree, alextait1]
+
+In this pattern, a parent container has a `role=”tablist”` with children elements that have a `role=”tab”`. These tabs are associated with elements that have a `role=”tabpanel”`, and contain the content for that tab. 
+
+{{ figure_markup(
+  image="role-tab-list.png",
+  alt="element with the tablist role",
+  caption='Tab list (`role="tablist"`) contains all of the tabs. ([Source](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html))',
+  description="TODO",
+  width=TODO,
+  height=TODO
+) }}
+
+{{ figure_markup(
+  image="role-tab.png",
+  alt="element with the tab role",
+  caption='“Nils Frahm” tab (`role="tab"`). ([Source](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html))',
+  description="TODO",
+  width=TODO,
+  height=TODO
+) }}
+
+{{ figure_markup(
+  image="role-tab-panel.png",
+  alt="element with the tabpanel role",
+  caption='Tab panel (`role="tabpanel"`) with content associated with the “Nils Frahm” tab. ([Source](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html))',
+  description="TODO",
+  width=TODO,
+  height=TODO
+) }}
+
+For desktop pages, 7.00% have at least one element with a `role=”tablist”` whereas there only 5.79% of pages have elements with a `role=”tab”` and 5.46% of pages have elements with a `role=”tabpanel”`. This suggests that the pattern may only be partially implemented. Even if there is dynamic rendering at play for some of the tab/tabpanel elements, the currently visible or first tab/tabpanel would theoretically be in the DOM on page load.
+
+##### Presentation
+
+When an element has been given a `role=”presentation”` its semantics are stripped away, for both the element it is assigned to and its required children. For example, tables and lists both have required children, so if the parent has a `role=”presentation”` this essentially cascades to the child elements, which will also have their semantics stripped. Removing an element’s semantics means that it is no longer that element in any capacity except for its visual appearance. For example, a list with a `role=”presentation”` will no longer communicate any information to a screen reader about the list structure.
+
+A common usage of this attribute is for `<table>` elements that have been used for layout rather than  for tabular data. We do not recommend using tables in this way. For layout, we have powerful CSS tools today such as flexbox and CSS grid. In general there are very few use cases where `role=”presentation”` is particularly helpful for assistive technology users, use this role sparingly and thoughtfully.
```

Comment:
```suggestion
A common usage of this attribute is for `<table>` elements that have been used for layout rather than  for tabular data. We do not recommend using tables in this way. For layout, we have powerful CSS tools today such as flexbox and CSS grid. In general there are very few use cases where `role="presentation"` is particularly helpful for assistive technology users, use this role sparingly and thoughtfully.
```